src/lib/modules/Loan/backoffice/LoanList/LoanListEntry.js

```diff
@@ -79,13 +80,19 @@ export class LoanListEntry extends Component {
               </List>
             </Grid.Column>
             <Grid.Column width={2} textAlign="center">
-              {deliveryMethod && (
+              {deliveryMethod ? (
                 <>
                   {delivery}{' '}
                   {deliveryMethod.iconClass && (
                     <Icon className={deliveryMethod.iconClass} />
                   )}
                 </>
+              ) : (
```

Comment:
I think there was a specified config for this, can't we use it?

Reply:
Used the config for the other occurrence of this, but here had to leave this piece of code because the logic of calculating `deliveryMethod` above (see line 21) is that it won't show the delivery method is the state of the loan is not `PENDING`. And for self-checkout we need to show it regardless.